CMake/cdat_modules/configobj_deps.cmake

```diff
@@ -0,0 +1 @@
+set(configobj_deps ${python_pkg} ${six_pkg})
```

Comment:
The only dependency listed in `six` https://github.com/DiffSK/configobj/blob/master/setup.py#L97
I'm uncertain if it requires setuptools (does setup.py expect this?)